cmd/omegabrr/main.go

```diff
@@ -59,7 +65,39 @@ func main() {
 
 	switch cmd := pflag.Arg(0); cmd {
 	case "version":
-		fmt.Fprintf(flag.CommandLine.Output(), "Version: %v\nCommit: %v\n", version, commit)
+		fmt.Printf("Version: %v\nCommit: %v\n", version, commit)
+
+		// get the latest release tag from brr-api
+		client := &netHTTP.Client{
+			Timeout: 10 * time.Second,
+		}
+
+		resp, err := client.Get(fmt.Sprintf("https://api.autobrr.com/repos/%s/%s/releases/latest", owner, repo))
```

Comment:
We could hardcode `owner` and `repo` here and on line 88 change to `no release found`